__init__.py

```diff
@@ -350,18 +357,21 @@ def calculate_anharmonic_thermo(self):
                 Z_mode = 1./(1.-np.exp(-e.real/(self.kT)))
                 ZPE = 0.5 * e.real
                 e_min_exitation = e.real
+                E_kT = e_min_exitation/self.kT
```

Comment:
Now E_kt (x) is a very local thing only defined here and used on the next line so it is fine to call it x.